src/CMakeLists.txt

```diff
@@ -57,6 +57,8 @@ add_library(OpenCL-objects OBJECT
 
 if (CLVK_UNIT_TESTING)
    target_compile_definitions(OpenCL-objects PUBLIC CLVK_UNIT_TESTING_ENABLED)
+   # unit.{cpp|hpp} is using "extern C" to avoid a mangling issue.
+   enable_language(C)
```

Comment:
```
$ cmake --version
cmake version 3.23.1
```
I don't really get why as well, but I have this error otherwise:
```
CMake Error in src/CMakeLists.txt:
  No known features for C compiler

  ""

  version .
```
```
cmake -B build -S . -G Ninja \
  -DCLVK_CLSPV_ONLINE_COMPILER=1 \
  -DCLVK_BUILD_TESTS=ON \
  -DCLVK_UNIT_TESTING=ON \
  -DCLVK_BUILD_STATIC_TESTS=OFF \
  -DCLVK_BUILD_CONFORMANCE_TESTS=ON \
  -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
  -DCLVK_VULKAN_IMPLEMENTATION=loader \
  -DCLVK_VULKAN_LOADER_DIR=external/Vulkan-Loader/ \
  -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Reply:
I don't really understand this but I'm fine with accepting the change as a temporary measure if it helps.

Reply:
thank you